.github/workflows/release.yml

```diff
@@ -13,6 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - name: Update version in pyproject.toml
+        run: |
+          VERSION=$(echo ${{ github.ref_name }} | sed 's/^v//')
```

Comment:
That did not work for 0.11.1

Reply:
I think the `VERSION` is missing an export?

Reply:
maybe